gap/PolygonalComplexes/properties.gd

```diff
@@ -129,6 +129,134 @@ DeclareProperty( "IsClosedSurface", IsTwistedPolygonalComplex and IsNotEdgeRamif
+#! faces, namely the tetradron.
+#! <Alt Only="TikZ">
+#!  \begin{tikzpicture}[vertexStyle, edgeStyle=nolabels, faceStyle]
+#!      \input{Image_Tetrahedron_Net.tex}
+#!  \end{tikzpicture}
+#! </Alt>
+#! And there is exactly one multi-tetrahedral sphere with 6 faces namely
+#! the double tetraedron which can be obtained by performing exactly one
+#! tetrahedral extension to the tetradron.
+#! <Alt Only="TikZ">
+#!      \input{Image_DoubleTetrahedron.tex}
+#! </Alt>
+#! @BeginExampleSession
+#! gap> IsMultiTetrahedralSphere(Tetrahedron());
+#! true
+#! gap> vof:=[[1,2,4],[2,3,4],[1,3,4],[1,2,5],[1,3,5],[2,3,5]];;
+#! gap> doubleTetra:=SimplicialSurfaceByVerticesInFaces(vof);
+#! simplicial surface (5 vertices, 9 edges, and 6 faces)
+#! gap> IsMultiTetrahedralSphere(doubleTetra);
+#! true
+#! @EndExampleSession
+#!
+#! As another example, consider the octahedron.
+#!
+#! @BeginExampleSession
+#! gap> IsMultiTetrahedralSphere(Octahedron());
+#! false
+#! @EndExampleSession
+#! @Returns true or false
+#! @Arguments complex
+DeclareProperty( "IsMultiTetrahedralSphere", IsTwistedPolygonalComplex );
+#! @EndGroup 
+
+#! @BeginGroup TetrahedralNumber
+#! @Description
+#! Return the number of tetrahedra that are used to construct the given 
+#! twisted polygonal complex. If <K>complex</K> is a multi-tetrahedral sphere,
+#! this number is 1 greater than the number of tetrahedral extensions that 
+#! have to be applied to the tetrahedron to obtain <K>complex</K>.
+#! If <K>complex</K> is not a multi-tetrahedral sphere, the function returns
+#! fail.
+#!
+#! As example, consider the tetrahedron, the double tetrahedron and 
+#! the multi-tetrahedral sphere which can be obtained by
+#! applying exactly two tetrahedral extensions to the tetrahedron.  
+#! @BeginExampleSession
+#! gap> doubleTetra:=TetrahedralExtension(Tetrahedron(),1);
+#! simplicial surface (5 vertices, 9 edges, and 6 faces)
+#! gap> multiTetra:=TetrahedralExtension(doubleTetra,2);
+#! simplicial surface (6 vertices, 12 edges, and 8 faces)
+#! gap> TetrahedralNumber(Tetrahedron());
+#! 1
+#! gap> TetrahedralNumber(doubleTetra);
+#! 2
+#! gap> TetrahedralNumber(multiTetra);
+#! 3
+#! @EndExampleSession
+#! @Returns positive integer or fail
+#! @Arguments complex
+DeclareAttribute( "TetrahedralNumber", IsTwistedPolygonalComplex );
+#! @EndGroup
+
+#! @BeginGroup TetrahedralType
+#! @Description
+#! Return the tetrahedral type of a given twisted polygonal complex or
+#! <K>fail</K>, if <K>complex</K> is not a multi tetrahedral sphere. 
+#! The tetrahedral type of a multi tetrahedral sphere is a tuple
+#! <K>[a0,...,ak]</K> satisfying the following property:
+#! <K>X</K> is a list of multi tetrahedral spheres of length <K>k</K> so that
+#! <K>X[1]=complex</K> and <K>X[k]</K> is either isomorphic to the tetrahedron 
+#! or the double tetrahedron. Furthermore <K>X[i]</K> is obtained by applying 
+#! tetrahedral reductions to all vertices with face degree 3 in <K>X[i-1]</K>.
```

Comment:
I don't understand how each X[i] can be isomorphic to an tetrahedron or a double tetrahedron and at the same time X[i] is obtained by applying tetrahedral reductions to all vertices with face degree 3 in X[i-1]. 
Does applying the tetrahedral reductions always result in a tetrahedral or a double tetrahedral?

Reply:
Only the last surface X[k] is isomorphic to the tetrahedron or the double tetrahedron. The other surfaces X[i] are obtained through tetrahedral reduction as described.

Reply:
Oh I see, my fault!